src/pages/STG_PanelRel.page

```diff
@@ -1,77 +1,121 @@
+                        </div>
+                    </div>
+                    <apex:outputPanel styleClass="slds-form-element" rendered="{!stgService.stgRel.Enable_Custom_Field_Sync__c}">
+                        <apex:outputLabel value="{!$ObjectType.Relationship_Sync_Excluded_Fields__c.Label}" for="slCFSF" styleClass="col-md-4 slds-form-element__label"
+                        />
+                        <div class="slds-form-element__control">
+                            <apex:outputText value="{!IF(excludedLabels == '', $Label.stgLabelNone, excludedLabels)}" rendered="{!isReadOnlyMode}" id="slCFSFO"
+                                html-aria-describedby="{!$Component.slCFSHelp}" styleClass="slds-form-element__static" />
+                            <apex:selectList value="{!listExcludedFields}" multiselect="true" size="5" rendered="{!isEditMode}" id="slCFSF" html-aria-describedby="{!$Component.slCFSFHelp}"
+                                styleClass="slds-select">
+                                <apex:selectOptions value="{!listSOSyncFieldOptions}" />
+                            </apex:selectList>
+                            <apex:outputPanel id="slCFSFHelp" layout="block">
+                                <apex:outputText styleClass="slds-form-element__help" value="{!$Label.stgHelpRelSyncFields}" />
+                            </apex:outputPanel>
+                        </div>
+                    </apex:outputPanel>
                 </apex:outputPanel>
-            </apex:outputPanel>
-        </div>
-        <div class="slds-grid slds-grid_align-center slds-grid_vertical-align-center slds-m-around_large">
-            <apex:commandButton id="saveRel" value="{!$Label.stgBtnSave}" status="statusLoad" action="{!saveSettings}" rendered="{!isEditMode}" immediate="false" rerender="form" styleClass="slds-button slds-button_small slds-button_brand" />
-            <apex:commandButton id="cancelRel" value="{!$Label.stgBtnCancel}" status="statusLoad" action="{!cancelEdit}" rendered="{!isEditMode}" immediate="true" rerender="form" styleClass="slds-button slds-button_small slds-button_neutral" />
-        </div>
+            </div>
+            <div class="slds-grid slds-grid_align-center slds-grid_vertical-align-center slds-m-around_large">
+                <apex:commandButton id="saveRel" value="{!$Label.stgBtnSave}" status="statusLoad" action="{!saveSettings}" rendered="{!isEditMode}"
+                    immediate="false" rerender="form" styleClass="slds-button slds-button_small slds-button_brand" />
+                <apex:commandButton id="cancelRel" value="{!$Label.stgBtnCancel}" status="statusLoad" action="{!cancelEdit}" rendered="{!isEditMode}"
+                    immediate="true" rerender="form" styleClass="slds-button slds-button_small slds-button_neutral" />
+            </div>
+        </apex:outputPanel>
     </apex:form>
+
+    <script>
+        const doesNotHaveAccess = {!hasAccess} === false;
```

Comment:
@psadie-sfdo What do you think about using `{!NOT(hasAccess)}` instead of `{!hasAccess} === false;` here?

Reply:
Sounds good to me! updated